cmd/upgrade_test.go

```diff
@@ -0,0 +1,56 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/exercism/cli/cli"
+	"github.com/stretchr/testify/assert"
+)
+
+type fakeCLI struct {
+	*cli.CLI
+}
+
+func (fc *fakeCLI) Upgrade() error {
+	fc.Version = fc.LatestRelease.Version()
+	return nil
+}
+
+func TestUpgrade(t *testing.T) {
+	fakeEndpoint := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		fmt.Fprintln(w, `{"tag_name": "v2.0.0"}`)
+	})
+	ts := httptest.NewServer(fakeEndpoint)
+	defer ts.Close()
+	cli.LatestReleaseURL = ts.URL
+
+	//disable printing to Stdout
+	out = ioutil.Discard
+
+	tests := []struct {
+		cv string
+		rv string
+	}{
+		{ // Calling upgrade on an outdated CLI initiates an upgrade.
+			cv: "1.0.0",
+			rv: "2.0.0",
+		},
+		{ // Calling upgrade on a CLI that's up-to-date  does nothing.
+			cv: "2.0.1",
+			rv: "2.0.1",
```

Comment:
Should this be 2.0.0 for the sake of clarity, since the endpoint returns 2.0.0.?